src/styles/_calendar.scss

```
@@ -0,0 +1,42 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  width: (($cell-size * 7) + ($cell-gap * 6)) + $padding * 2;
+  padding: $padding;
+
+  @each $day, $n in $days {
+    &--start-day--#{$day} &__day:first-child {
+      margin-left: ($cell-size + $cell-gap) * $n;
+    }
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length--#{$index} :nth-child(n + #{$index +1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    @include square($cell-size);
+    @extend %display;
+    border: 1px solid black;
+    background-color: #eee;
+    cursor: pointer;
+    transition: 0.5s;

```

```suggestion
    transition: background color 0.5s, transform 0.5s;

```
And move 0.5s to the variable